src/screens/Movie/Movie.tsx

```diff
@@ -49,23 +50,31 @@ const Movie = ({ match, location }: RouteComponentProps<MovieRouteParams>): JSX.
 
   // Media
   const { isLoading, error, data: item } = useMedia(id);
-  const itemRequiresSubscription = item?.requiresSubscription !== 'false';
   useBlurImageUpdater(item);
   const { data: trailerItem } = useMedia(item?.trailerId || '');
   const { data: playlist } = useRecommendedPlaylist(recommendationsPlaylist || '', item);
 
-  const isFavorited = useFavoritesStore((state) => !!item && state.hasItem(item));
+  // AccessModel & entitlement
+  const mediaOffer = useMemo(() => (item?.productIds && filterCleengMediaOffers(item.productIds)) || undefined, [item]);
+  useEntitlement(mediaOffer);
```

Comment:
Can this be removed?

Reply:
It re-appeared after a merge. Thanks, fixed!

Reply:
It re-appeared after a merge. Thanks, fixed!